docs/src/guide/installation.md

```diff
@@ -1,15 +1,23 @@
 # Installation
 
-Similar to [installing any other registered package](https://julialang.github.io/Pkg.jl/v1/getting-started/)
-in Julia just run from a Julia REPL:
-```
-] add DFTK
+In case you have not yet installed Julia, first do so by
+[downloading the Julia binaries](https://julialang.org/downloads/)
+and following the [Julia installation instructions](https://julialang.org/downloads/platform/).
+At least **Julia 1.4** is required for DFTK.
+
+Afterwards you can install DFTK
+[like any other package](https://julialang.github.io/Pkg.jl/v1/getting-started/)
+in Julia. For example run in your Julia REPL terminal:
+```julia
+using Pkg
+pkg"add DFTK"
```

Comment:
I think we want to encourage people to use `]`, no? That's more idiomatic and more pleasant.

Reply:
Actually it's less clear for beginners what this means in my experience. A person who knows Julia will not read this anyway.

Reply:
I'd write "type `]` to get into package mode, then type `add DFTK`". I thought we had that at some point

Reply:
Yes we did. But people in other packages are moving away from this, too, see  http://docs.juliaplots.org/latest/install/ or https://github.com/JuliaPy/PyCall.jl which I would say represent amongst the first Julia packages people will encounter.

Reply:
I am ok with `Pkg.add("DFTK")` as well if you prefer that.

Reply:
Ok for Pkg.add then. pkg"" is just too magical. It's just annoying to have to `import Pkg`.